### PR TITLE
GH-1616: tracker list breakdown detailed view

### PR DESCRIPTION
**Pull request by wlycdgr**
Show additional Cliqz details for each tracker in the detailed view: number of cookies blocked / fingerprints wiped by Cliqz for this tracker, if any.

~~Does not implement the ad portion of the ticket, as discussed.~~  Now implements the ad portion of the ticket also.

Note that I haven't been able to find a fingerprint hit in the wild to confirm that it is passed through correctly from the background. However, I've confirmed that the fingerprint readout as well as the cookie readout displays correctly when passed dummy data on the front end.

* [X] Have you followed the guidelines in [CONTRIBUTING.md](../CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you added an explanation of what your changes do?
* [X] Does your submission pass tests?
* [X] Did you lint your code prior to submission?

### CODE REVIEW

**Comment by sammacbeth** [comment]
I managed to get a fingerprint shown,  looks good!

Screen Shot 2019-06-11 at 13 00 12
Screen Shot 2019-06-11 at 14 04 55

File: src/classes/FoundBugs.js [hunk 1]
```diff
@@ -276,6 +276,16 @@ class FoundBugs {
 		return apps_arr;
 	}
 
+	getAppsById(tab_id) {
```
Comment (IAmThePan) [inline src/classes/FoundBugs.js]: Please add a documentation comment for this function.

Comment (wlycdgr) [reply src/classes/FoundBugs.js]: Added

**Comment by wlycdgr** [comment]
Ads portion of GH-1616 implemented

**Comment by wlycdgr** [comment]
Fix bug where panel would not display if a bug id returned by getGhosteryStats did not map to an app id in our PanelData trackerList

**Pull request merged**